ports/touhoucirno/port.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 3,
+  "name": "touhoucirno.zip",
+  "items": [
+    "Touhou - Cirno in Streets of Frog.sh",
+    "touhoucirno"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Touhou: Cirno in Streets of Frog",
+    "porter": [
+      "Ganimoth"
+    ],
+    "desc": "Yokai Mountain Megacity is under attack by the criminal Frog Syndicate, and it's up to the strongest undercover fairy police officer to stop it! Punch, kick and parry your way through an onslaught of pizza-throwing knights, pop culture references, frogs and breakable crates in this smash hit that'll make your fingers bleed! Battle your way through 10 stages of frogs, fairies and ghost cosplayers in this frantic arcade action game!",
+    "desc_md": null,
+    "inst": "Go to the game's page https://yaru.itch.io/touhou-streets-of-frog, download the game and place the FrogStreet__v1_1__itchio.zip into gamedata folder.",
```

Comment:
These underscores are going to cause havoc on the website, maybe add an escaped version in `"inst_md"` ?

Reply:
Or I will just tell them to place the zip file there, that should do.